policy-controller/grpc/src/lib.rs

```diff
@@ -320,5 +342,198 @@ fn to_authz(
         networks,
         labels,
         authentication: Some(authn),
+        metadata: None, // TODO fill
+    }
+}
+
+fn to_http_route(
+    name: &String,
+    route: &HttpRoute,
+    srv: &InboundServer,
+    cluster_networks: &[IpNet],
+) -> proto::HttpRoute {
+    let metadata = Metadata {
+        kind: Some(metadata::Kind::Resource(metadata::Resource {
+            group: "gateway.networking.k8s.io".to_string(),
+            kind: "HTTPRoute".to_string(),
```

Comment:
it's kind of unfortunate that we have to create a new instance of these strings for _every_ `HTTPRoute`, since they're always the same...but we can figure out a solution for that later, if it's worth bothering with. unfortunately, we probably can't get `prost` to let us use `Cow<'static, str>` for proto types...

Reply:
Yeah, I don't think there's any way around this without changing prost.